pages/content/amp-dev/documentation/guides-and-tutorials/integrate/story-interactive.md

```diff
@@ -0,0 +1,71 @@
+
+In terms of actual implementations, it's a good idea to give users control over the theme and chip-style of the elements, and ideally also allow them to control the accent color and prompt alignment. Variables such as prompt text and prompt background are interesting for users since they allow prompt backgrounds to be gradients, and they allow prompt text to be colored to match better the story design cues and improve contrast.
+
+Tools can go above and beyond to give extra customization options for creators. Many interesting designs can be implemented with the `chip-style="transparent"` attribute. For inspiration check out the [amp-story-interactive-results Codepen](https://codepen.io/mszylkowski/pen/abNWzdb) which contains creative designs for transparent styles. Those require extra CSS rules to be modified for the elements, such as the actual element's background, border (and border-radius), box-shadow, backdrop-filter, etc. When enabling extra options, keep in mind use-cases for them, as many CSS rules will not work on the elements (eg: font-family).
+
+### Integrating amp-story-interactive-results
+
+The interactive experience of gathering the answers to polls or quizzes might be complex for first-time users to understand, so tools should provide some guidance on that regard. Percentage results automatically link to quizzes while polls require extra configuration, but either way, users should get a warning when adding a result if no quizzes or poll were added to the story.
+
+For quiz results, it's a good idea to allow users to set the thresholds visually (eg: with a slider split up into many sections) and be able to edit/preview all the states separately.
+
+For categorical results (linked to polls) users should have a way of easily linking the `results-category` attributes to the states of the amp-story-interactive-results element. Confusion can happen if users manually mistype the category, resulting in the options not being properly linked to the result state, so it's a good idea to either give feedback to the user when the category in an option doesn't match the other categories defined on the story, or to provide users with dropdowns to link options to already defined categories.
+
+More templated tools can also create entire workflows to make stories with polls and results, but this approach will not fit most WYSIWYG editors.
+
+### Backends
+
+Tools can support their own backends to aggregate the data, but it is recommended to use one of the already available endpoints in the [documentation](https://amp.dev/documentation/components/amp-story-interactive?format=stories). Setting up a backend is a significant effort and might require forward support as more types of experiences are rolled out.
+
+If using one of the provided backends, tools should be able to fetch the state of the interactive data directly through the same endpoint as provided in the backend. This information can be either gathered server side or on the frontend.
```

Comment:
I think all we should document here is how to poll the Google hosted backend to fetch a Story's poll/quizz results.